backend/core/src/shared/hmi-data.ts

```diff
@@ -0,0 +1,220 @@
+import { Unit } from "../units/entities/unit.entity"
```

Comment:
@jaredcwhite , just want to confirm, that nothing was changed here, except for pulling these pieces out of unit-transformations to get the file size down under 400 lines, correct?

Reply:
Commented too soon, looks like `amiPercentagesAcrossUnits`, `findAmiValueInChart`, and `setupHmiHeaders` was pulled out